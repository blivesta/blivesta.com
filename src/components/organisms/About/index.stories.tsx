import React from 'react';
// import styled from 'styled-components';
import { storiesOf } from '@storybook/react';
import About from '.';

storiesOf(`organisms|About`, module).add(`default`, () => <About />);
