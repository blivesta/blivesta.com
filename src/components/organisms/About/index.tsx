import * as React from 'react';

import ContentBlock from '../../molecules/ContentBlock';
import Title from '../../atoms/Title';
import SubTitle from '../../atoms/SubTitle';
import Lists from '../../atoms/Lists';
import P from '../../atoms/P';

const About = () => {
  return (
    <ContentBlock id="about" title="About">
      <Title>
        Design to
        <br />
        accelerate growth
      </Title>
      <P lang="en">
        We think strategically about design and help companies and brands growth. We conduct research concept
        formulation, visual UI, web application and site development as a whole, and we are also working on analysis,
        proposal and improvement in operation.
      </P>
      <P lang="ja" last justify>
        デザインを戦略的に考え、企業やブランドをグロースさせるためのお手伝いをしています。リサーチ・コンセプト策定、ビジュアル・UI、Webアプリ・サイト開発を一環して行い、運用でも分析・提案・改善に取り組んでいます。
      </P>
      <SubTitle>Capabilitties:</SubTitle>
      <Lists
        isNotPipeline
        items={[
          {
            title: 'User interface design',
          },
          {
            title: 'User exprerience design',
          },
          {
            title: 'Brand Identity design',
          },
          {
            title: 'Web development',
          },
          {
            title: 'Softwear development',
          },
          {
            title: 'Project/Product management',
          },
        ]}
      />
    </ContentBlock>
  );
};

export default About;
