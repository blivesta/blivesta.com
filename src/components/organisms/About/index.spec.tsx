import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import About from '.';

describe('About', () => {
  it('snapshot', () => {
    const tree = shallow(<About />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
