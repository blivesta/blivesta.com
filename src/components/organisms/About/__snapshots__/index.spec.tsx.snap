// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About snapshot 1`] = `
<ContentBlock
  id="about"
  title="About"
>
  <styled.h1>
    Design to
    <br />
    accelerate growth
  </styled.h1>
  <P
    lang="en"
  >
    We think strategically about design and help companies and brands growth. We conduct research concept formulation, visual UI, web application and site development as a whole, and we are also working on analysis, proposal and improvement in operation.
  </P>
  <P
    justify={true}
    lang="ja"
    last={true}
  >
    デザインを戦略的に考え、企業やブランドをグロースさせるためのお手伝いをしています。リサーチ・コンセプト策定、ビジュアル・UI、Webアプリ・サイト開発を一環して行い、運用でも分析・提案・改善に取り組んでいます。
  </P>
  <styled.h3>
    Capabilitties:
  </styled.h3>
  <Lists
    isNotPipeline={true}
    items={
      Array [
        Object {
          "title": "User interface design",
        },
        Object {
          "title": "User exprerience design",
        },
        Object {
          "title": "Brand Identity design",
        },
        Object {
          "title": "Web development",
        },
        Object {
          "title": "Software development",
        },
        Object {
          "title": "Project/Product management",
        },
      ]
    }
  />
</ContentBlock>
`;
